lib/features/email/data/network/email_api.dart

```diff
@@ -23,9 +23,9 @@ import 'package:jmap_dart_client/jmap/mail/email/submission/email_submission.dar
 import 'package:jmap_dart_client/jmap/mail/email/submission/email_submission_id.dart';
 import 'package:jmap_dart_client/jmap/mail/email/submission/envelope.dart';
 import 'package:jmap_dart_client/jmap/mail/email/submission/set/set_email_submission_method.dart';
+import 'package:model/model.dart';
```

Comment:
You should import specifically the class to be used. Avoid importing the whole module.